hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestSnapshotDeletingService.java

```diff
@@ -189,10 +206,16 @@ public void testMultipleSnapshotKeyReclaim() throws Exception {
     // verify the cache of purged snapshot
     // /vol1/bucket2/bucket2snap1 has been cleaned up from cache map
     assertEquals(2, om.getOmSnapshotManager().getSnapshotCacheSize());
+    //clean data
+    //    // cleaning up the data
```

Comment:
nit: can you please fix the duplicate comments?

Reply:
Done. thanks.